recipes/azure-mgmt/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.20.1" %}
+
+package:
+  name: azure-mgmt
+  version: {{ version }}
+
+source:
+  fn: azure-mgmt-{{ version }}.zip
+  url: https://pypi.io/packages/source/a/azure-mgmt/azure-mgmt-{{ version }}.zip
+  md5: 8a678938c5e3f809392fe23e3653e83f
+  
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - azure-mgmt-compute ==0.20.1
+    - azure-mgmt-network ==0.20.1
+    - azure-mgmt-resource ==0.20.1
+    - azure-mgmt-storage ==0.20.0
+  run:
+    - python
+    - azure-mgmt-compute ==0.20.1
+    - azure-mgmt-network ==0.20.1
+    - azure-mgmt-resource ==0.20.1
+    - azure-mgmt-storage ==0.20.0
```

Comment:
One thing you might consider to make life easier here is to just use `{{ version }}` for all of these as they seem to many seem to share the same version as the package itself. This might make updating a little nicer.

Also, I think ` =={{ version}}` is the same as ` {{ version}}`.

If there are a couple that always lag, we might be able to come up with some programmatic to describe that.

All of this is totally up to you and not required. Just thinking of ways to make updating this package easier.